dev/conductor/ui/test/widgets/steps_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+import 'package:conductor_ui/widgets/progression.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:platform/platform.dart';
+
+void main() {
+  group('Progession steps and substeps workflow', () {
+    testWidgets(
+        'All substeps of the current step must be checked before able to continue to the next step',
+        (WidgetTester tester) async {
+      await tester.pumpWidget(
+        StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return MaterialApp(
+              home: Material(
+                child: Column(
+                  children: <Widget>[
+                    MainProgression(
+                      releaseState: null,
+                      stateFilePath:
+                          defaultStateFilePath(const LocalPlatform()),
```

Comment:
Using the `LocalPlatform()` makes the test non-hermetic (this may also be why the windows framework-test-misc is failing). Prefer a FakePlatform().

Reply:
Also, it looks like in this test, this would only be used for the error message. In that case, you can just hard code a path in. While you're at it, you can add an expectation to verify that path is rendering.